### ISSUE TITLE
keyboard navigation of headers to get the sort order

### ISSUE THREAD

**Issue by wallym** [wenzhixin/bootstrap-table#6204]
### Description

I'm currently integrating the bootstrap table into an internal site that has to be section 508 accessible for US government usage.  I am getting hit on the sort ordering is not accessible for keyboard users.  I've tried to google to find a solution, but alas, nothing pops up that sounds helpful.  any ideas are appreciated.

TIA

### Example(s)

The following is my table definition:
            <table id="tblSearchResults"  data-loading-template="loadingTemplate"
                    data-toolbar=".toolbar" data-pagination="true" data-search="true" data-show-search-clear-button="true"
                    data-virtual-scroll="true" data-show-columns="true" 
                    class="table table-sm table-striped table-hover table-borderless ">
                      <thead>
                        <tr class="table-light">
                          <th data-field="DocketNo" data-sortable="true" data-formatter="linkFormatter" class="priority-1">Docket No</th>
                          <th data-field="RecipientName" data-sortable="true" data-formatter="recipientNameLengthFormatter" class="priority-2">Recipient Name</th>
                          <th data-field="CaseOpen" data-formatter="dateFormat" data-sortable="true" class="priority-3">Received</th>
                          <th data-field="CaseResolved" data-formatter="dateFormat" data-sortable="true" class="priority-4">Resolved</th>
                          <th data-field="processingAge" data-sortable="true" class="priority-5">Case Age</th>
                          <th data-field="StageDescription" data-sortable="true" class="priority-5">Stage</th>
                          <th data-field="LeadInvestigatorFormalName" data-sortable="true" class="priority-5">Lead Inv.</th>
                          <th data-field="LeadAttorneyFormalName" data-sortable="true" class="priority-5">Lead Att.</th>
                          <th data-field="ResoTypeDescription" data-sortable="true" class="priority-5">Resolution Type</th>
                        </tr>
                      </thead>
                </table>

My css is defined as:
@media screen and (max-width: 1350px) and (min-width: 1250px) {
    .priority-5 {
        display: none;
    }
}

@media screen and (max-width: 1250px) and (min-width: 1150px) {
    .priority-5 {
        display: none;
    }

    .priority-4 {
        display: none;
    }
}

@media screen and (max-width: 1150px) and (min-width: 1050px) {
    .priority-5 {
        display: none;
    }

    .priority-4 {
        display: none;
    }

    .priority-3 {
        display: none;
    }
}

@media screen and (max-width: 1050px) {
    .priority-5 {
        display: none;
    }

    .priority-4 {
        display: none;
    }

    .priority-3 {
        display: none;
    }

    .priority-2 {
        display: none;
    }

}

**Comment by UtechtDustin** [comment]
Please provide us an example using our [online editor](https://live.bootstrap-table.com/).

**Comment by UtechtDustin** [comment]
I close this issue, because we got no example from you within 7 days.
If you still need help please provide us an example and we will reopen the issue, thanks!
